File: dreamedge/fizzbuzz.php
<?php
for($i=1; $i<100; $i++){
  if($i%15 == 0)
  {
    echo "fizzbuzz\n";
  }
  else if($i%3 == 0)
  {
    echo "fizz\n";
  }
  else if($i%5 == 0)
  {
    echo "buzz\n";
  }
  else
  {
    echo $i, "\n";
  }
}
?>

File: sakou-k/fizzbuzz.c
#include <stdio.h>
int main(int ac,char** av)
{
	int i;
	for( i = 1;i < 255;i++){
		if( i % 3 == 0 && i % 5 == 0){
			printf("fizz buzz\n");
		} else if ( i % 3 == 0 ){
			printf("fizz\n");
		} else if ( i % 5 == 0 ){
			printf("buzz\n");
		} else {
			printf("%d\n",i);
		}
	}
	return 0;
}
